src/error.rs

```diff
@@ -22,7 +22,7 @@ pub enum DecryptionErrorCode {
     /// The sender device is not cross-signed.
     UnsignedSenderDevice,
     /// The sender's identity is unverified, but was previously verified.
-    SenderIdentityPreviouslyVerified,
+    SenderIdentityVerificationViolation,
```

Comment:
this looks like a change we need to mention in the changelog

Reply:
Thanks, fixed in f147076